docs/src/scs/pki.md

```diff
@@ -25,6 +25,36 @@ HSM can be also used to perform cryptographic operations such as digital signing
 
 One example of affordable HSM solutions is YubiHSM, developed by Yubico.
 
+### HSM Variants for consideration
+
+The following HSM Solutions are considered for Ghaf project:
+
+  - [YubiHSM2](https://www.yubico.com/fi/product/yubihsm-2/ "YubiHSM2")
+  - [NitrokeyHSM2](https://shop.nitrokey.com/shop/product/nkhs2-nitrokey-hsm-2-7 "NitrokeyHSM2")
+  - [SoftHSMv2](https://github.com/opendnssec/SoftHSMv2 "SoftHSMv2")
+
+The following table aims to provide feature comparison of the proposed solutions. Since the feature list is quite extensive, the table is limited to the features that are either planned to be used in Ghaf or might benefit the project in the future.
```

Comment:
The following table **provides** a feature comparison of the proposed solutions.

Let's make this sentence with a new line and as a note using ">" (https://www.markdownguide.org/basic-syntax/#blockquotes-1):

> Since the feature list is quite extensive, the table is limited to the features that are either planned to be used in Ghaf or might benefit the project in the future.